.github/workflows/wheel.yaml

```diff
@@ -0,0 +1,106 @@
+name: Build and upload to PyPI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build_wheels:
+    name: Build wheels for cp${{ matrix.python }}-${{ matrix.platform_id }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # linux-64
+          - os: ubuntu-latest
+            python: 38
+            platform_id: manylinux_x86_64
+          - os: ubuntu-latest
+            python: 39
+            platform_id: manylinux_x86_64
```

Comment:
It'd be lovely if this section was condensed a little bit, since most of it seems to be fairly repetitive.

I assume that this is set up follow some standard naming of wheels? Something like
```
      matrix:
        include:
          - os: [ubuntu-latest, macos-latest]  # Windows missing zlib
          - python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

with aliasing to what's currently `platform_id` etc. (not sure how to do this in YAML - I assume it's possible) would be much more compact and easier to maintain.